src/pages/TeachersUnite/IntroSchoolPrincipalPage.tsx

```diff
@@ -0,0 +1,153 @@
+import Str from 'expensify-common/lib/str';
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx/lib/types';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import useEnvironment from '@hooks/useEnvironment';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import * as LoginUtils from '@libs/LoginUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import TeachersUnite from '@userActions/TeachersUnite';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {LoginList} from '@src/types/onyx';
+
+type IntroSchoolPrincipalFormData = {
+    firstName: string;
+    lastName: string;
+    partnerUserID: string;
+};
+
+type IntroSchoolPrincipalPageOnyxProps = {
+    loginList: OnyxEntry<LoginList>;
+};
+
+type IntroSchoolPrincipalPageProps = IntroSchoolPrincipalPageOnyxProps;
+
+function IntroSchoolPrincipalPage(props: IntroSchoolPrincipalPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {isProduction} = useEnvironment();
+
+    /**
+     */
+    const onSubmit = (values: IntroSchoolPrincipalFormData) => {
+        const policyID = isProduction ? CONST.TEACHERS_UNITE.PROD_POLICY_ID : CONST.TEACHERS_UNITE.TEST_POLICY_ID;
+        TeachersUnite.addSchoolPrincipal(values.firstName.trim(), values.partnerUserID.trim(), values.lastName.trim(), policyID);
+    };
+
+    /**
+     * @returns {Object} - An object containing the errors for each inputID
```

Comment:
Remove types from JSDOC:
```suggestion
     * @returns - An object containing the errors for each inputID
```

Reply:
@blazejkustra 

Fixed